### ISSUE TITLE
hmac panic on mac os x in 4.34.0_1

### ISSUE THREAD

**Issue by MrSpecterist** [v2fly/v2ray-core#744]
## What version of V2Ray are you using?
4.34.0_1 (brew version)

## What's your scenario of using V2Ray?
Vmess websocket

## What problems have you encountered?
it's not working and initiating at all

## Please attach error logs here

```
V2Ray 4.34.0 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.16 darwin/amd64)
A unified platform for anti-censorship.
2021/03/06 20:46:38 [Info] v2ray.com/core/main/jsonem: Reading config: /usr/local/etc/v2ray/config.json
panic: crypto/hmac: hash generation function does not produce unique values

goroutine 134 [running]:
crypto/hmac.New(0xc002ee6c60, 0xc000408000, 0x18, 0x18, 0x18, 0x18)
	crypto/hmac/hmac.go:143 +0x370
v2ray.com/core/proxy/vmess/aead.KDF(0xc000400030, 0x10, 0x10, 0xc002ee6e20, 0x1, 0x1, 0xc002e66280, 0x1a286b8, 0xc00008fd60)
	v2ray.com/core/proxy/vmess/aead/kdf.go:13 +0x12f
v2ray.com/core/proxy/vmess/aead.KDF16(...)
	v2ray.com/core/proxy/vmess/aead/kdf.go:22
v2ray.com/core/proxy/vmess/encoding.(*ClientSession).DecodeResponseHeader(0xc000400000, 0x1b00b20, 0xc001ffa360, 0x0, 0x0, 0x0)
	v2ray.com/core/proxy/vmess/encoding/client.go:195 +0xd6
v2ray.com/core/proxy/vmess/outbound.(*Handler).Process.func3(0x0, 0x0)
	v2ray.com/core/proxy/vmess/outbound/outbound.go:161 +0x16b
v2ray.com/core/common/task.OnSuccess.func1(0x0, 0x0)
	v2ray.com/core/common/task/task.go:12 +0x2f
v2ray.com/core/common/task.Run.func1(0xc001ff60a8, 0xc000012600, 0xc001ff8330)
	v2ray.com/core/common/task/task.go:28 +0x2b
created by v2ray.com/core/common/task.Run
	v2ray.com/core/common/task/task.go:27 +0x152
panic: crypto/hmac: hash generation function does not produce unique values

goroutine 119 [running]:
crypto/hmac.New(0xc002eeac60, 0xc000196e28, 0x18, 0x18, 0x18, 0x18)
	crypto/hmac/hmac.go:143 +0x370
v2ray.com/core/proxy/vmess/aead.KDF(0xc0008d4030, 0x10, 0x10, 0xc002eeae20, 0x1, 0x1, 0xc0008aa048, 0x0, 0x0)
	v2ray.com/core/proxy/vmess/aead/kdf.go:13 +0x12f
v2ray.com/core/proxy/vmess/aead.KDF16(...)
	v2ray.com/core/proxy/vmess/aead/kdf.go:22
v2ray.com/core/proxy/vmess/encoding.(*ClientSession).DecodeResponseHeader(0xc0008d4000, 0x1b00b20, 0xc0008ac3c0, 0x0, 0x0, 0x0)
	v2ray.com/core/proxy/vmess/encoding/client.go:195 +0xd6
v2ray.com/core/proxy/vmess/outbound.(*Handler).Process.func3(0x0, 0x0)
	v2ray.com/core/proxy/vmess/outbound/outbound.go:161 +0x16b
v2ray.com/core/common/task.OnSuccess.func1(0x0, 0x0)
	v2ray.com/core/common/task/task.go:12 +0x2f
v2ray.com/core/common/task.Run.func1(0xc0008a8088, 0xc0008b25a0, 0xc0008aa240)
	v2ray.com/core/common/task/task.go:28 +0x2b
created by v2ray.com/core/common/task.Run
	v2ray.com/core/common/task/task.go:27 +0x152
```

**Comment by Loyalsoldier** [comment]
This is a known issue when building V2Ray with Go v1.16, that has been fixed in V2Ray v4.35.0+. Please wait for it to be released and updated on Homebrew. Or you can download pre-built binary from https://github.com/v2fly/v2ray-core/releases for now.

**Comment by Loyalsoldier** [comment]
V2Ray in Homebrew has been updated to the latest v4.35.1
